docs/platforms/javascript/guides/fastify/index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Fastify
+description: "Learn about using Sentry with Fastify."
+---
```

Comment:
Here we need to add the category information - have a look at the Node guide to see how it is set.

Reply:
it worked 🚀